yt_dlp/extractor/teamcoco.py

```diff
@@ -76,126 +124,157 @@ class TeamcocoIE(TurnerBaseIE):
         }, {
             'url': 'http://teamcoco.com/israel/conan-hits-the-streets-beaches-of-tel-aviv',
             'only_matching': True,
-        }, {
-            'url': 'https://conan25.teamcoco.com/video/ice-cube-kevin-hart-conan-share-lyft',
-            'only_matching': True,
-        }
+        },
     ]
-    _RECORD_TEMPL = '''id
-        title
-        teaser
-        publishOn
-        thumb {
-          preview
-        }
-        tags {
-          name
-        }
-        duration
-        turnerMediaId
-        turnerMediaAuthToken'''
-
-    def _graphql_call(self, query_template, object_type, object_id):
-        find_object = 'find' + object_type
-        return self._download_json(
-            'https://teamcoco.com/graphql', object_id, data=json.dumps({
-                'query': query_template % (find_object, object_id)
-            }).encode(), headers={
-                'Content-Type': 'application/json',
-            })['data'][find_object]
 
     def _real_extract(self, url):
-        display_id = self._match_id(url)
+        display_id = self._match_id(url).replace('/', '_')
```

Comment:
display_id can have `/`. You don't have to change this if this is ur preference though

Reply:
can video_id have `/`? display_id used as a fallback if the "real" video_id can't be extracted

Reply:
Technically yes. But I do agree it's a bad idea